<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpWindow</class>
 <widget class="QMainWindow" name="UdpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>本地地址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_comboLocalAddr">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0.0.0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>本地端口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_editLocalPort">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>远程地址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_editRemoteAddr">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>远程端口:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_editRemotePort">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="m_editOutput">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="m_editSend">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnSend">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actBind"/>
   <addaction name="m_actAbort"/>
   <addaction name="m_actClear"/>
   <addaction name="separator"/>
   <addaction name="m_actExit"/>
  </widget>
  <action name="m_actBind">
   <property name="icon">
    <iconset resource="Udp.qrc">
     <normaloff>:/images/bind.bmp</normaloff>:/images/bind.bmp</iconset>
   </property>
   <property name="text">
    <string>绑定</string>
   </property>
   <property name="toolTip">
    <string>绑定</string>
   </property>
  </action>
  <action name="m_actAbort">
   <property name="icon">
    <iconset resource="Udp.qrc">
     <normaloff>:/images/abort.bmp</normaloff>:/images/abort.bmp</iconset>
   </property>
   <property name="text">
    <string>解绑</string>
   </property>
   <property name="toolTip">
    <string>解绑</string>
   </property>
  </action>
  <action name="m_actClear">
   <property name="icon">
    <iconset resource="Udp.qrc">
     <normaloff>:/images/clear.bmp</normaloff>:/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="m_actExit">
   <property name="icon">
    <iconset resource="Udp.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Udp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_actExit</sender>
   <signal>triggered()</signal>
   <receiver>UdpWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
